Traceback (most recent call last):
  File "H:\GitHub\FRITS-Backup\main.py", line 9, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
